jobs/ci-run/build/builddqlite.yml

```diff
@@ -94,6 +98,41 @@
       - host-src-command:
           src_command: !include-raw: ../scripts/snippet_make-dqlite-build.sh
 
+- job:
+    name: cross-build-dqlite-ppc64el
```

Comment:
Rename to `build-dqlite-ppc64el`. I don't think we need to expose that it's doing cross-compilation at this level (we're leaking implementation details here).